src/include/platform/PlatformManager.h

```diff
@@ -332,6 +337,40 @@ inline CHIP_ERROR PlatformManager::InitChipStack()
     return err;
 }
 
+/**
+ * @brief
+ *   Shuts down and cleans up the main objects in the CHIP stack.
+ *   This DOES NOT stop the chip thread or event queue from running.
+ *
+ */
```

Comment:
Any reason for this move? Seems to me that the PR is equivalent without the move

Reply:
I reordered some functions to make more sense and be more consistent. I figure now is as good a time as any, not many others are working in this file. If that's a problem I can restore any original order, but it may be cumbersome.

Reply:
I undid all the reordering improvements in this file.